projects/batfish-common-protocol/src/main/java/org/batfish/common/plugin/SyncTestrigsPlugin.java

```diff
@@ -0,0 +1,17 @@
+package org.batfish.common.plugin;
+
+import java.util.Map;
+
+public abstract class SyncTestrigsPlugin extends CoordinatorPlugin {
```

Comment:
Can you add significant javadoc here explaining what this plugin is for, how it's used, etc?